ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml

```diff
@@ -0,0 +1,155 @@
+    </p>
+
+    <h3>
+        What about food?
+    </h3>
+    <p>
+        No food will be provided.
+    </p>
+
+    <h3>
+        Who wins?
+    </h3>
+    <p>
+        See the <a href="./Rules">Rules</a> page for more information.
+    </p>
+
+    <h3>
+        How do we know if we've solved a puzzle?
+    </h3>
+    <p>
+        Each of the puzzles has a simple final answer, either a single word or a short phrase. In some puzzles you are explicitly told how to get the final answer. In others, figuring out how to extract the final answer is part of the puzzle. <strong>You enter puzzle answers into the website and are immediately told whether they are correct</strong>. If you solve a puzzle and you get a long phrase it is probably a clue to a shorter answer. Try entering the phrase into the website. If the word/phrase is recognized and you get a "Keep going" or similar message, that means you entered a valid partial answer and are on the right track.
+    </p>
+
+    <h3>What is a "Metapuzzle"? </h3>
+    <p>A metapuzzle is a special puzzle that takes the progress you've made from solving other puzzles and combines them in some way. It is not necessary to solve all other puzzles before solving the final metapuzzle. The metapuzzle will be clearly labeled for Intern Puzzleday 2019.</p>
+
+    <h3>
+        How do we solve puzzles that don't have instructions?
+    </h3>
+    <p>
+        The implicit instruction for all puzzles is "Figure out the final answer." Puzzles without instructions are either classic puzzle forms, perhaps with a twist, or puzzles where the challenge is figuring out how to solve the puzzle. In the latter case, the key is usually to look for patterns or common bonds between the parts of the puzzle. For example, are many of the words spelled wrong? Have all the vowels been left out? Sometimes there may be a hint hidden in the puzzle.
+    </p>
+
+    <h3>
+        What's the best way to win?
+    </h3>
+    <p>
+        The best way to win is to solve puzzles quickly so that you get to the final metapuzzle. The best way to accomplish this is with teamwork. Many teams prefer to work individually on the easier puzzles and work as a group on the harder puzzles. If you get stuck, ask for help or switch to another puzzle. Make sure you keep track of which puzzles you've already solved and confirmed using the answer submission system.
+    </p>
+
+    <h3>
+        What's the best way to lose?
+    </h3>
+    <p>
+        Not having fun. If you're not having fun working on a particular puzzle, switch to another one. Don't worry, you're unlikely to run out of puzzles.
+    </p>
+
+    <h3>
+        What if we're really stuck?
+    </h3>
+    <p>
+        If you don't understand how to solve a puzzle, contact puzzle control. We may give you a hint or clarify the instructions. Aside from that, there are enough puzzles that you probably won't be stuck on all of them.
```

Comment:
Should this say to use the links on the submission page to contact puzzle control?